### ISSUE TITLE
Issues with the latest version 1.2.2

### ISSUE THREAD

**Issue by sheltonpaul89** [ForbesLindesay/sync-rpc#2]
Getting the below error,

```
_node_modules/sync-rpc/lib/index.js:80
    throw new Error(
    ^

Error: Timed out waiting for sync-rpc server to start (it should respond with "pong" when sent "ping"):

[eval]:1
var c=require('net').connect(59542,'127.0.0.1',()=>{c.end("ping\r\n").pipe(process.stdout)});
                                                                     ^

TypeError: Cannot read property 'pipe' of undefined
    at Socket.require.connect ([eval]:1:70)
    at Socket.g (events.js:286:16)
    at emitNone (events.js:86:13)
    at Socket.emit (events.js:185:7)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1067:10)


    at waitForAlive (node_modules/sync-rpc/lib/index.js:80:11)
    at start (node_modules/sync-rpc/lib/index.js:40:3)
    at sendMessage (node_modules/sync-rpc/lib/index.js:109:17)
    at createClient (node_modules/sync-rpc/lib/index.js:149:27)
    at Object.<anonymous> (node_modules/sync-request/lib/index.js:16:14)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)_
```

**Comment by ForbesLindesay** [comment]
I've made a change that will hopefully fix this (try v1.2.3 which was just released). Make sure you're also on node version 8 as sync-rpc doesn't support older versions.

**Comment by smartm0use** [comment]
Hello, sorry for this post but this place is the one in which I can ask for it.
Anyone can help me about the timeout sync-rpc error with package that have it among dependencies?

https://github.com/vercel/pkg/issues/204#issuecomment-1173093090
